notebooks/multiclass/UnorderedMulticlass.ipynb

```diff
@@ -0,0 +1,505 @@
+      "text/plain": [
+       "<Figure size 1080x360 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Display the results of the explainability evaluation for both models\n",
+    "# The table displays the distributions over the number of features that must be changed to alter predictions\n",
+    "explainability_df = construct_scores_dataframe(scores('explainability', results))\n",
+    "display(explainability_df)\n",
+    "\n",
+    "%matplotlib inline\n",
+    "\n",
+    "def plot_histogram(axs, model_id):\n",
+    "    num_features = list(range(1, 11))\n",
+    "    columns = [f\"Num features ({i})\" for i in num_features]\n",
+    "    percentages = explainability_df.loc[model_id, columns].values\n",
+    "    \n",
+    "    axs.bar(num_features, percentages)\n",
+    "    axs.set_xticks(num_features)\n",
+    "    axs.set_xlabel('Number of features', fontsize=15)\n",
+    "    axs.set_title(f\"Model {model_id}\", fontsize=15)\n",
+    "    axs.set_ylabel('Distributions', fontsize=15)\n",
+    "    axs.set_ylim(top=100)\n",
+    "\n",
+    "def plot_explainability_histograms():\n",
+    "    labels = explainability_df['context'].values\n",
+    "    \n",
+    "    fig, (axs1, axs2) = plt.subplots(1, 2, figsize=[15,5])\n",
+    "    fig.suptitle('Explainability results by model', fontsize=20)\n",
+    "    \n",
+    "    plot_histogram(axs1, labels[0])\n",
+    "    plot_histogram(axs2, labels[1])\n",
+    "    \n",
+    "    fig.tight_layout()\n",
+    "    plt.show()\n",
+    "    \n",
+    "plot_explainability_histograms()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "logistic Accuracy:  1.0\n",
+      "SVM Accuracy:  1.0\n"
+     ]
```

Comment:
both accuracy as 1 doesn't seem correct?

Reply:
Good observation. I found this [SO post](https://stackoverflow.com/questions/36967126/why-do-i-get-good-accuracy-with-iris-dataset-with-a-single-hidden-node) where the first answer mentions a high accuracy is common because there is high correlation between two features and the outcome class (links to a file in the UCI archive verifying this).

Is this something that you think would raise some questions? (would you recommend commenting in notebook about this?)

Reply:
the code is absolutely fine, its just that the dataset is very simplistic with heavy correlation with the outcome columns. If you increase the test/train split ration to 0.4, accuracy comes down to 98.33, but i did it just to make sure. All looks cool to me.